pcdsdevices/attenuator.py

```diff
@@ -1071,6 +1072,114 @@ class AT2L0(FltMvInterface, PVPositionerPC, LightpathInOutMixin):
     # Summary for lightpath view
     num_in = Cpt(InternalSignal, kind='hinted')
     num_out = Cpt(InternalSignal, kind='hinted')
+    
+    """check for errors and return a string indicating any errors verbally"""
+    def _check_errors(signals: SignalToValue) ->  str:
+        errors = []
+        #sort out .motor from .motor.plc signals
+        for sig, value in signals.items():
+            if ("motor" in sig.name) ^ ("plc" in sig.name):
+                value=sig.metadata.get('severity').value
```

Comment:
The formatting indicates you didn't get install pre-commit. I commented in your other PR but will mention it here in case you read it first: https://confluence.slac.stanford.edu/display/PCDS/How+to+add+a+device+to+pcdsdevices#Howtoaddadevicetopcdsdevices-Pre-commit